tests/Kernel.php

```diff
@@ -31,7 +31,6 @@ protected function configureContainer(ContainerBuilder $container, LoaderInterfa
         } else {
             $loader->load(__DIR__.'/config/config_php7.yaml');
         }
-        $loader->load(__DIR__.'/../config/services.xml');
```

Comment:
This was wrong, because the `MeilisearchBundle` loads the services, so this was just overrode always what the bundle defined, that means what we set in `MeilisearchExtension` was lost and many tests were failing because the configuration was not properly injected to the `SearchService`.